### PR TITLE
Added configuration copy postinstall script

### PR DESCRIPTION
**Pull request by hari-govind**
Addresses issue #19 . Created config/copyConfig.js.

### CODE REVIEW

File: config/copyConfig.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+ // Script to copy config.sample.js to config.js
+
+const fs = require('fs')
+
+try{
+var sample_config_file = fs.readFileSync('config/config.sample.js', 'utf8');
```
Comment (sakshamsaxena) [inline config/copyConfig.js]: It's preferred to use `path.join(__dirname, 'config.sample.js')`. This script will behave differently when invoked from different directories, so it's best to be as absolute as we can.

File: config/copyConfig.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+} catch (err) {
+    if (err.code === 'ENOENT') {
```
Comment (sakshamsaxena) [inline config/copyConfig.js]: You can simply `console.error(err.message)` here, don't bother with handling different error cases yourself.

File: config/copyConfig.js (hunk 1 above)
Comment (sakshamsaxena) [reply config/copyConfig.js]: Instead of `try...catch`, use the async `fs.readFile`

**Comment by sakshamsaxena** [comment]
You can create an async promise chain or call `fs.writeFile` in `fs.readFile`'s callback. Both are correct. Promises are ideal however, but not necessary for this script.

**Comment by hari-govind** [comment]
@sakshamsaxena,  replaced with fs.writeFile and fs.readFile

File: config/copyConfig.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+const fs = require('fs')
+const path = require('path')
+const util = require('util');
+
+const readFile = util.promisify(fs.readFile);
+const writeFile = util.promisify(fs.writeFile);
+
+readFile(path.join(__dirname, 'config.sample.js'))
+    .then((content) => {
+       writeFile(path.join(__dirname, 'config.js'), content)
```
Comment (sakshamsaxena) [inline config/copyConfig.js]: Excellent! Minor change here :
Chaining promises is done by `return`ing another promise from inside a `then`. So you should `return writeFile(...)` here instead, shift the `then` after it, and that would also eliminate the use of the extra `catch()`
Give it a try! :smile:

Comment (hari-govind) [reply config/copyConfig.js]: So like this ?
```javascript
readFile(path.join(__dirname, 'config.sample.js'))
    .then((content) => {
       return writeFile(path.join(__dirname, 'config.js'), content)
            .then(() => {
                console.log('Sample configuration sucessfully copied.');
            })
```

Comment (sakshamsaxena) [reply config/copyConfig.js]:
```javascript
readFile(path.join(__dirname, 'config.sample.js'))
    .then((content) => {
        console.log('inside first then')
        return writeFile(path.join(__dirname, 'config.js'), content)
    })
    .then(() => {
        console.log('Sample configuration sucessfully copied.');
    })
```

**Comment by hari-govind** [comment]
@sakshamsaxena , added it :)

**Pull request merged**